drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -232,20 +232,21 @@ struct bcm2835_isp_fmt *find_format_by_fourcc(unsigned int fourcc,
 					      struct bcm2835_isp_node *node)
 {
 	struct bcm2835_isp_fmt_list *fmts = &node->supported_fmts;
-	struct bcm2835_isp_fmt *fmt;
+	const struct bcm2835_isp_fmt *fmt;
 	unsigned int i;
 
 	for (i = 0; i < fmts->num_entries; i++) {
-		fmt = &fmts->list[i];
+		fmt = fmts->list[i];
```

Comment:
This change isn't obvious from the commit message.

Reply:
"The list of supported formats was taking a copy of entries from this table.
This is unnecessary, therefore allocate an array of pointers instead of
an array of entries."

Seeing as it was all messing with struct bcm2835_isp_fmt I wasn't too worried over splitting it into multiple commits.